demos/index.html

```diff
@@ -0,0 +1,7 @@
+<h1>Glimmer Demos</h1>
```

Comment:
pretty sure this file should be dropped, at least until we have something we want to demo ;)